client/src/partials/patients/edit/edit.html

```diff
@@ -3,9 +3,18 @@
 
     <ol class="headercrumb">
       <li class="static">{{ "TREE.HOSPITAL" | translate }}</li>
-      <li class="title">{{ "PATIENT_EDIT.PAGE_TITLE" | translate }}</li>
+      <li class="static">{{ "PATIENT_RECORDS.TITLE" | translate }}</li>
+      <li class="title">{{ PatientEditCtrl.medical.name }}</li>
     </ol>
 
+    <!-- ng-show to stop flickering on load -->
```

Comment:
No. `ng-cloak` is concerned with hiding elements whilst angular is loading - this ensure that the edit tag is only displayed if the context is available (in this case waiting on a GET request for the patient's name)